tools/crawl_summary/crawl_report.py

```diff
@@ -0,0 +1,186 @@
+from collections import Counter
+import datetime
+import os
+import inspect
+import json
+
+STR_DATETIME_FORMAT = "%d_%m_%y_%H_%M_%S"
+
+
+class CrawlReport:
+    def __init__(self) -> None:
+        path_to_dir = os.sep.join(inspect.getfile(self.__class__).split(os.sep)[:-1])
```

Comment:
Yes, I am not sure. This is initialized in the article_crawler.py - so I think the parameter would be passed there instead for this file specifically, maybe I can make a setter for the dir path, though the generate_report also expects this path to be in a specific place.

I could make both the `article_crawler` and the `generate_report` take a `CrawlReportPath` in case we want to change it.

Reply:
I decided to go with the environment variables, I think it's working!